manim/scene/scene_file_writer.py

```diff
@@ -499,13 +523,20 @@ def open_partial_movie_stream(self, file_path=None):
             self.video_container = video_container
             self.video_stream = stream
 
+            self.queue = Queue()
+            self.writer_thread = Thread(target=self.listen_and_write, args=())
+            self.writer_thread.start()
```

Comment:
I'm not sure if I like the idea of starting a new thread inside `__init__` (`__init__` should be a cheap initialization whenever possible), what do you think about putting it into a separate method `start_writer_thread` and calling it right after instantiation?

Reply:
Sounds good to me! I just wanted to clarify that I am creating/starting the thread in `open_partial_movie_stream`, rather than in the `SceneFileWriter` constructor.

Reply:
Ah that's my bad - it's fine then.
Would this mean that a new thread is being opened every `play` call?

Reply:
Did some testing, I think it's fine as it is right now. Marking this as resolved :)